iris_gpubench/nvidia_nsight/setup_nsights.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+# Set up system
+cat /proc/sys/kernel/perf_event_paranoid
+
+sudo sh -c 'echo 2 >/proc/sys/kernel/perf_event_paranoid'
+
+sudo sh -c 'echo kernel.perf_event_paranoid=2 > /etc/sysctl.d/local.conf'
```

Comment:
We shouldn't change this globally because it will persist after reboot (which potentially has security implications)
The `echo` to `/proc` should persist until the next reboot, and if they re-run the benchmark it will re-run this setup AFAIK?